src/dodal/devices/athena/panda/tables.py

```diff
@@ -0,0 +1,149 @@
+from collections import deque, namedtuple
```

Comment:
Should: A lot of the work in here seems closely coupled to the generic panda itself, e.g. the helper functions for building up tables. Why are they in here and not with the panda in `ophyd-async`?

Reply:
We've discussed and think FlyingPanda should be in ophyd-async alongside Panda as you suggest, so will resolve this and move there.